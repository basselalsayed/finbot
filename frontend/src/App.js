import React, { Component } from 'react';
import Greeting from './components/greeting/Greeting'
import Header from './components/header/Header'
import Header2 from './components/header/Header2'
import Chatbot from './components/chatbot/Chatbot'
import ThemeSwitcher from './ThemeSwitcher'
import Draggable from 'react-draggable';
// eslint-disable-next-line
import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link
} from "react-router-dom";
import './App.css';


export default class App extends Component {

  render() {
  return (
    <Router>
      <div className="App">
        <Header2 />
        <Greeting />
        <Chatbot />
      </div>
    </Router>
  );
  }
}


