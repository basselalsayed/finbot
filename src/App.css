.App {
  text-align: center;
  /* display: flex;
  flex-direction: column;
  align-items: flex-start;
  width: 100%; */
}
/* .App>*:first-child {
  align-self: stretch;
} */

.card-text {
  color: white;
  padding: 10px;
  border-radius: 25px;
  width: 75%;
  display: flexbox;
}

.card-body {
  overflow-y: auto;
  height: 500px;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

.typing {
  display: block;
  width: 60px;
  height: 40px;
  background-color: #BDBDBD;
  margin-left: 20px;
  border-radius: 15px;
  display: flex;
  justify-content: center;
  align-items: center;
}

.circle {
  display: block;
  height: 10px;
  width: 10px;
  border-radius: 50%;
  background-color: #757575;
  margin: 3px;
  animation: bounce 1000ms ease-in-out;
}

.circle:nth-child(1){
  animation-delay: 0ms
}
  
.circle:nth-child(2) {
  animation-delay: 333ms
}

.circle:nth-child(3) {
  animation-delay: 666ms
}
