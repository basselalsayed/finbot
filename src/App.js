import React, { Component } from 'react';
// import Greeting from './components/greeting/Greeting'
// import Header from './components/header/Header'
import Header2 from './components/header/Header2'
import Chatbot from './components/chatbot/Chatbot'
// import ThemeSwitcher from './ThemeSwitcher'
// import Draggable from 'react-draggable';
// eslint-disable-next-line
import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link
} from "react-router-dom";
import './App.css';


export default class App extends Component {

  render() {
  return (
    <Router>
    <link
      rel="stylesheet"
      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
      crossorigin="anonymous"
    />
      <div className="App">
        <Header2 />
        {/* <Greeting /> */}
        <Chatbot />
      </div>
    </Router>
  );
  }
}


