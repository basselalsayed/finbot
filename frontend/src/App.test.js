import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';
import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link
} from "react-router-dom";

test('renders learn react link', () => {
  // const { getByText } = render(<App />);
  // // const linkElement = getByText(/learn react/i);
  // expect('hello').toBeInTheDocument();

  const component = renderer.create(
    <App />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
