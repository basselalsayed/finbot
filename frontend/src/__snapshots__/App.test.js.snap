// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <header
    style={
      Object {
        "background": "#333",
        "color": "#fff",
        "padding": "10px",
        "textAlign": "center",
      }
    }
  >
    <h1>
      Finbot
    </h1>
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "#fff",
          "textDecoration": "none",
        }
      }
    >
      Home
    </a>
  </header>
  
</div>
`;
